djvm/src/main/kotlin/net/corda/djvm/references/Member.kt

```diff
@@ -20,17 +21,19 @@ typealias MethodBody = (EmitterModule) -> Unit
  * @property exceptions The names of the exceptions that the member can throw.
  * @property value The default value of a field.
  * @property body One or more handlers to replace the method body with new byte-code.
+ * @property runtimeContext Local runtime "state" objects for each [Emitter].
```

Comment:
This data class should represent a member of a class, correct? Why do you need to introduce the emitter "state"? What is exactly the state in the runtime context? Are emitter immutable?

Reply:
The emitters are considered to be immutable, yes. And they are also reused across the entire sandbox and so they cannot be allowed to remember anything. (I am fixing a memory leak here.)

I've added this _optional_ runtime state to `Member` so that emitters like `DisallowCatchingBlacklistedExceptions` and `HandleExceptionUnwrapper` can remember what they learnt from `MethodVisitor.visitLabel()` when they're asked to handle `MethodVisitor.visitTryCatchBlock()`. And then this state is _automatically_ discarded when the `MethodVisitor` completes.